Pod/Classes/DictionaryConvenience.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  DictionaryConvenience.swift
+//  Pods
+//
+//  Created by Matt Thomas on 5/20/16.
+//
+//
+
+import Foundation
+
+public func += <Key, Value> (inout left: [Key: Value], right: [Key: Value]) {
```

Comment:
I attempted to make these parameters optional and when I tried to use the operator, I got errors saying I needed to unwrap the optional use use it with the operator.

*I* don’t have any needs for optional parameters. Maybe when somebody does, they can investigate this.